.travis.yml

```diff
@@ -2,13 +2,22 @@
 language: node_js
 node_js:
   - "6"
```

Comment:
I've removed one of the node versions as this is what was causing logs to never show on Travis.

Reply:
Node 8 is currently the LTS version. Perhaps we should remove `6` instead of `8`?

Reply:
So the ember-cli adding .travis.yml actually recommends 4 but that fails to install due to peer dependencies not being met.

Reply:
Let's merge this now and come back to it later in https://github.com/ebryn/ember-model/issues/472